lib/components/primitive-components/Group/Group.ts

```diff
@@ -198,7 +207,9 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
         {
           method: "POST",
           body: JSON.stringify({
-            input_circuit_json: this.root!.db.toArray(),
+            input_circuit_json: pcbAndSourceCircuitJson,
+            subcircuit_id:
+              this.componentName === "Board" ? "" : this.subcircuit_id,
```

Comment:
When the `Board` is sent to autoroute, no filter should be done on the circuit_json as the whole elements are required. This case is handled in the backend here - https://github.com/tscircuit/registry-api/pull/146